Gifski/ConversionViewController.swift

```diff
@@ -16,12 +16,14 @@ final class ConversionViewController: NSViewController {
 	private lazy var timeRemainingEstimator = TimeRemainingEstimator(label: timeRemainingLabel)
 
 	private var conversion: Gifski.Conversion!
+	private var gifData: Data?	
 	private var progress: Progress?
 	private var isRunning = false
 
-	convenience init(conversion: Gifski.Conversion) {
+	convenience init(_ conversion: Gifski.Conversion, _ gifData: Data?) {
```

Comment:
hmm this seems strange that we still need the `Conversion` object even though we had the gif data already 

I know that we need this object in the ConversionCompleted vc, but maybe we could try to think about better structure of this?

ideally I'd opt for an `enum` on this init that would tell us whether we need a gif conversion from the video or do we actually have the gif data already, e.g.:
```swift
enum GifSource { // <-- hate this name so this is only an example
    case conversionNeeded(Conversion)
    case gifDataAvailable(Data)
}
```

this would make it really clear in code with what data we work